test/org/sagebionetworks/bridge/services/StudyServiceTest.java

```diff
@@ -332,7 +332,15 @@ public void problematicHtmlIsRemovedFromTemplates() {
     @Test
     public void adminsCanChangeSomeValuesResearchersCannot() {
         study = TestUtils.getValidStudy(StudyServiceTest.class);
-        setStudyDefaults(study);
+        study.setStudyIdExcludedInExport(true);
+        study.setEmailVerificationEnabled(true);
+        study.setExternalIdValidationEnabled(false);
+        study.setExternalIdRequiredOnSignup(false);
+        study.setEmailSignInEnabled(false);
+        study.setPhoneSignInEnabled(false);
+        study.setReauthenticationEnabled(false);
+        study.setAccountLimit(0);
+        study.setAutoVerificationPhoneSuppressed(false);
```

Comment:
It's probably not necessary to include autoVerificationPhoneSuppressed in this test (set or assert), or in changeStudyDefaults().